ONSdigital/dp-image-importer
# Add ImageUploadedEvent handler

### What

Added an event handler for `ImageUploadedEvent` kafka events. Includes…
- Removal of `dp cli` generated `api` package (not needed by event handlers)
- Using vault client for getting AND putting files (unless `ENCRYPTION_DISABLED=true`)
- Using S3 clients with/without psk
- Adding and updating image variants on the Image API, with ServiceAuth
- Updating to go 1.15
- Adding a local test producer (which calls the local image API rather than directly creating  a kafka event itself)

### How to review

- Ensure code is sensible and tests are written
- Check comments and code for any bed references to StaticFilePublisher etc (code copied over)
- Check for unneeded API related code (the service initially used the `API` project in code generation)

### Who can review

Anyone but me
